plugins/sentry/api-report.md

```diff
@@ -112,11 +120,13 @@ export type SentryIssue = {
 export const SentryIssuesWidget: ({
   entity,
   statsFor,
+  tableOptions,
   variant,
   query,
 }: {
   entity: Entity;
-  statsFor?: '' | '14d' | '24h' | undefined;
+  statsFor: '24h' | '14d' | '';
+  tableOptions: Options<object>;
```

Comment:
is there possible a more specific/narrow type than `object`?

Reply:
@jhaals I am new to typing and didn't see `RowData` as an exported member nor see anything added to it, I assumed it was just an object. The type is strict though, if I put in anything that isn't a table option it will complain. Any suggestions?
https://github.com/material-table-core/core/blob/master/types/index.d.ts#L331

Reply:
Ah I see that RowData is private, make it `Options<never>` in that case.
What types of customisations are you interested in doing specifically? I wonder if it make sense to expose those customisations explicitly as this might make it hard to understand how and why this is used.

Reply:
Well, it started with the fact I wanted the default amount of rows for the Sentry table to be 10, then I quickly realized it would be better to open up all of the options as then if I needed to make other modifications to the table in the future I could just by passing in the whole object rather than making another pr.